### PR TITLE
Docsp 8976

### PR DESCRIPTION
**Pull request by madelinezec**
IN PROGRESS

### CODE REVIEW

**Comment by madelinezec** [comment]
I left our orig dockerfile lines commented out in case after we test prod, we want to go back to only using snooty release

**Comment by madelinezec** [comment]
IN PROGRESS

File: worker/jobTypes/githubJob.js [hunk 1]
```diff
@@ -213,8 +230,29 @@ class GitHubJobClass {
...
+                await fs.writeFileSync(
+                    `${snootyDir}/gatsby-config.js`,
+                    gatsbyConfigFileContents.content, {
+                        encoding: 'utf8',
+                        flag: 'w'
+                    }
+                );
+            } else {
+                console.log(
+                    'ERROR: gatsby config file does not exist in /configfiles directory on meta branch.'
```
Comment (skerschb) [inline worker/jobTypes/githubJob.js]: Won't this just get overwritten when snooty is rsync'ed over during make next-gen-html?

https://github.com/mongodb/docs-worker-pool/blob/meta/makefiles/Makefile.docs-node#L32

We could copy the file local to the repo but not within snooty dir, and then split out the next-gen-html target to a publish target that has the step that copies this config to repoDir/snooty?

Also we should make a failure to download this file a fatal error -- return control to the prodDeploy job and fail on the job.

File: worker/jobTypes/githubJob.js [hunk 2]
```diff
@@ -213,8 +230,29 @@ class GitHubJobClass {
         // check if need to build next-gen
         if (this.buildNextGen()) {
             commandsToBuild[commandsToBuild.length - 1] = 'make next-gen-html';
+           }
+         //overwrite gatsby config file
+        if (currentJob.payload.jobType === 'productionDeploy'){
```
Comment (skerschb) [inline worker/jobTypes/githubJob.js]: This conditional needs a check for next gen as well, perhaps wrap in the previous.

**Pull request closed without merging**